pontoon/base/migrations/0065_fix_projectlocale_permissions.py

```diff
@@ -0,0 +1,45 @@
+from django.db import migrations
+
+from pontoon.base.models import ProjectLocale
+from pontoon.base.signals import assign_group_permissions, create_group
+
+
+def fix_projectlocale_permissions(apps, schema_editor):
+    for project, locale in [
+        ("common-voice", "nqo"),
```

Comment:
Missed a few earlier, apparently.

```suggestion
        ("common-voice", "cdo"),
        ("common-voice", "dar"),
        ("common-voice", "nqo"),
        ("common-voice", "shn"),
```